internal/utils/kruize/kruize_api.go

```diff
@@ -45,14 +46,25 @@ func Create_kruize_experiments(experiment_name string, k8s_object []map[string]i
 	}
 	res, err := http.Post(url, "application/json", bytes.NewBuffer(payload))
 	if err != nil {
-		return nil, fmt.Errorf("error Occured while creating experiment: %v", err)
+		log.Errorf("error Occured while creating experiment: %v", err)
```

Comment:
We need to `return` from here if there is any issue while calling kruize API.

Reply:
Won't this exit the function?

Reply:
Ah, for other scenarios, you mean

Reply:
When kruize return "performance profile already exist" at that time this is not triggered. err is only set when there is any issue in making http request. for example - kruize is down.